content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -93,6 +109,20 @@ the `startingCSV` field of the subscription is not possible when using skiprange
 The above warning means that skiprange by itself is useful for teams who are not interested in
 supporting directly installing versions within a given range.
 
+#### Example for SkipRange
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
```

Comment:
Same comment as above - there are differing indentation levels for the examples documented here.

Reply:
Worked on all the indentation issue.